appcelerator-archive/amp
# [v0.6.x] Account command - verify

ref #679 

Implements the `verify` command on the CLI. User enters the code they were sent on their verification email and then enters their username and creates their password which are validated by `validators.go`. The validations will be:

- Code must be exactly 8 characters long.
- Username must match an existing username
- Password must be at least 8 characters long and contain 3 unique sets of characters (lowercase, uppercase, number, symbol).

Currently, only the code and password can be verified.

To test
```
make install
amp pf start --local
amp account verify
```

This will be iterated on when the backend is created.
